### ISSUE TITLE
safe dog does not suggest nrs version on subname removal

### ISSUE THREAD

**Issue by davidpbrown** [maidsafe/safe-api#564]
Removing a subname leaves both dog and cat giving error without first suggesting the NRS version.

So, as detailed below, I created a subname and removed it as safe://subname.testrm
but checking dog for safe://testrm just gives an error - which might be appropriate but obscures that v=0 does still exist.. and without that perhaps breaks perpetual web interests.

The error currently is
```
$ safe dog safe://testrm
[2020-05-31T14:55:35Z ERROR safe] safe-cli error: [Error] ContentError - Sub name not found in NRS Map Container
```

where I would expect perhaps instead should suggest that NRS is at v=1 because that tempts user to consider that this will work; so, that is the error should follow what detail does exist.
```
$ safe dog safe://testrm?v=0
```

The setup was
```
$ safe nrs create 'subname.testrm' --link safe://hnyynyimynxodk579sj63csz51sqqjkobktb7ukgt1bb4nhostquxqakfwbnc?v=0
New NRS Map for "safe://subname.testrm" created at: "safe://hnyydysh5ccudgdrsgchwqc7rcp67su1o8edgqstipht1k8hosknrgdb4obqh"
+  subname.testrm  safe://hnyynyimynxodk579sj63csz51sqqjkobktb7ukgt1bb4nhostquxqakfwbnc?v=0 
$ safe nrs remove subname.testrm
NRS Map updated (version 1): "safe://subname.hnyydysh5ccudgdrsgchwqc7rcp67su1o8edgqstipht1k8hosknrgdb4obqh"
-  subname.testrm  safe://hnyynyimynxodk579sj63csz51sqqjkobktb7ukgt1bb4nhostquxqakfwbnc?v=0 
$ safe nrs remove subname.testrm
NRS Map updated (version 1): "safe://subname.hnyydysh5ccudgdrsgchwqc7rcp67su1o8edgqstipht1k8hosknrgdb4obqh"
-  subname.testrm  safe://hnyynyimynxodk579sj63csz51sqqjkobktb7ukgt1bb4nhostquxqakfwbnc?v=0 
```

**Comment by dan-da** [comment]
At first glance, I think present behavior is probably correct, but probably best to hear from @bochaco.

**Comment by bochaco** [comment]
It's correct behavior, trying to fetch `safe://subname.testrm` also gives you an error as it's virtually removed, since you are trying to resolve latest version of a subname/name from that NRS container which is version 1, and the subname/name is gone, so that's correct.

As per `safe://testrm`, it doesn't resolve because it was created as an alias to the subname, if you wanted to create it as a direct link to the same location you should create it with the `--direct` flag:
```
$ safe nrs create subname.testrm --direct --link safe://hnyynyimynxodk579sj63csz51sqqjkobktb7ukgt1bb4nhostquxqakfwbnc?v=0
```

**Comment by davidpbrown** [comment]
The error is correct for what it is now
.. but it's not a complete answer because it doesn't suggest the NRS value.

I'm expecting the history should be apparent in some form.. perpetual web.

So, a response that notes NRS=1 ahead of the error, is better. That tempts that calling on v=0 will see the old Alias. If the NRS version count is higher there's no guessing what went directly before without some dull iteration through all possible v=.

**Comment by bochaco** [comment]
That would imply looking up which version that subname existed, it could be a few or many versions back as you say, so we cannot do that and start looking up for it back in history, we don't even know if it was ever there so I don't think we should try to suggest that either.

**Comment by davidpbrown** [comment]
I'm not suggesting that the subname version is declared.. rather that the current NRS version is stated. Exercise for the reader to followup if they want the history.

Just seems odd to suggest nothing here and not noting the NRS version.. because surely if another update was made the NRS would iterate and become visible again?

**Comment by bochaco** [comment]
Right, showing the current version should be possible, yeah, that sounds good to me.
